com.carpeludum.KegaFusion.desktop

```
@@ -0,0 +1,10 @@
+[Desktop Entry]
+Name=Kega Fusion
+Comment=Sega 8 and 16-bit consoles emulator
+Type=Application
+MimeType=application/x-sega-cd-rom;application/x-sega-pico-rom;application/x-genesis-rom;application/x-genesis-32x-rom;application/x-sg1000-rom;application/x-sms-rom;application/x-gamegear-rom;
+Categories=Audio;AudioVideo;Graphics;GTK;Player;Video;Viewer;
```

All this categories seems irrelevant. Is it a left-over?
```suggestion
Categories=Game;Emulator;
```

---

I think the comment above still applies.